deliver/lib/deliver/detect_values.rb

```diff
@@ -67,5 +69,17 @@ def find_platform(options)
         options[:platform] = 'osx'
       end
     end
+
+    def verify_languages!(options)
+      languages = options[:languages]
+      return unless languages
+
+      all_languages = Spaceship::Tunes.client.available_languages
+      diff = languages - all_languages
```

Comment:
the only downside to this approach is that I don't think it does case-insensitive compares before returning the difference of the two arrays.
Case matters so much when it comes to language codes in ITC.